src/seml/utils/__init__.py

```diff
@@ -605,12 +633,12 @@ def wrapper(*args, **kwargs):
                 logging.warning(warning.format(num_calls=num_calls))
             return f(*args, **kwargs)
 
-        return wrapper
+        return wrapper  # type: ignore
```

Comment:
This should be fixable?

Reply:
technically no, but a cast is more explicit than a type ignore